### ISSUE TITLE
Support for Geometry Interfaces 

### ISSUE THREAD

**Issue by pmdartus** [jsdom/jsdom#2716]
Spec: [Geometry Interfaces Module Level 1](https://drafts.fxtf.org/geometry)

**Comment by valtism** [comment]
I would like to see support for this. Currently tests are throwing because `DOMRect` constructor is not defined.

**Comment by craigkovatch** [comment]
Yup, we get random failures on unit tests around our company when someone accidentally tickles this as well.

**Comment by EliteScientist** [comment]
Bump. I keep running into this and have to setup custom environments to define a DOMRect class. It would be nice if jsdom supported it.

**Comment by atomictangerine** [comment]
+1 to what @valtism said. Would be awesome to be able to test components that use DOMRect using jest. Thanks!

**Comment by radesai21** [comment]
+1 to what @valtism said as well. This has become a blocker in our unit testing due to errors from `DOMRect` not being defined

**Comment by domenic** [comment]
https://twitter.com/slicknet/status/782274190451671040

**Comment by JonasDoe** [comment]
> https://twitter.com/slicknet/status/782274190451671040

Well, that doesn't realy cover it completely: writing such a comment in an issue like this will keep it from becoming stale (and possibly auto-closed), and it might raise the ticket's priority by showing there's much interest in it.
